receiver/oracledbreceiver/metadata.yaml

```diff
@@ -109,6 +109,51 @@ metrics:
       value_type: int
       input_type: string
     unit: "{reads}"
+  oracledb.physical_reads_direct:
+    description: Number of reads directly from disk, bypassing the buffer cache
+    enabled: false
+    sum:
+      aggregation_temporality: cumulative
+      monotonic: true
+      value_type: int
+      input_type: string
+    unit: "{reads}"
+  oracledb.physical_read_io_requests:
+    description: Number of read requests for application activity
+    enabled: false
+    sum:
+      aggregation_temporality: cumulative
+      monotonic: true
+      value_type: int
+      input_type: string
+    unit: "{requests}"
+  oracledb.physical_writes:
+    description: Number of physical writes
+    enabled: true
```

Comment:
Please set enabled false when adding a new metric and move to make enable it by default in a breaking change, as a separate PR.